src/test/java/io/confluent/connect/hdfs/HdfsSinkConnectorTestBase.java

```diff
@@ -73,6 +72,9 @@
     props.put(PartitionerConfig.LOCALE_CONFIG, "en");
     props.put(PartitionerConfig.TIMEZONE_CONFIG, "America/Los_Angeles");
 
+    props.put(HdfsSinkConnectorConfig.TOPIC_REGEX_CAPTURE_GROUP_CONFIG, "(.*)[\\.\\-](.*)");
```

Comment:
Do these belong in the test base class?
Maybe we can scope these to just the relevant test classes? idk, i'm kinda 50/50 on this.
It does mean that we're not testing with this config disabled, which probably isn't good.

I think you can keep the refactor for topicsDir as a Map, but just disable the config options.

Reply:
I put this in the test base class because it is the easiest way to "integration" test this config to make sure it works